pkg/cli/mirror/mirror.go

```diff
@@ -245,6 +245,12 @@ func (o *MirrorOptions) Run(cmd *cobra.Command, f kcmdutil.Factory) (err error)
 
 	var mapping image.TypedImageMapping
 	var meta v1alpha2.Metadata
+
+	// Three mode options
+	mirrorToDisk := len(o.OutputDir) > 0 && o.From == ""
```

Comment:
I wish these modes are something more explicit and verifiable, not something quirky and implicit like this. However, this PR may not be the right place to address this. Perhaps a follow-up PR in the future.